lib/plugins/standalone.js

```diff
@@ -40,10 +42,16 @@ module.exports = class Standalone {
   }
 
   async upgrade() {
+    const commandRunStartTime = Date.now();
+    mainProgress.notice('Resolving latest standalone version', { isMainEvent: true });
     const tagName = await standaloneUtils.resolveLatestTag();
     const latestVersion = tagName.slice(1);
     if (latestVersion === currentVersion) {
-      this.serverless.cli.log('Already at latest version');
+      const message = 'Already at latest version';
```

Comment:
I wouldn't introduce those variables, as when we will remove the `legacy` calls, they will leave unnecessary verbosity.

I think it's fine in this case to keep given messages mentioned twice, it'll pay off in v3 branch